opentech/apply/activity/tests/test_messaging.py

```diff
@@ -642,3 +650,51 @@ def test_slack_lead_change(self):
         data = json.loads(responses.calls[0].request.body)
         self.assertIn(str(user), data['message'])
         self.assertIn(str(project), data['message'])
+
+    @override_settings(
+        SLACK_DESTINATION_URL=target_url,
+        SLACK_DESTINATION_ROOM=target_room,
+    )
+    @responses.activate
+    def test_slack_applicant_update_payment_request(self):
+        responses.add(responses.POST, self.target_url, status=200, body='OK')
+
+        project = self.source_factory()
+        payment_request = PaymentRequestFactory(project=project)
+        applicant = ApplicantFactory()
+
+        self.adapter_process(
+            MESSAGES.UPDATE_PAYMENT_REQUEST,
+            adapter=self.slack(),
+            user=applicant,
+            source=project,
+            related=payment_request,
+        )
+
+        self.assertEqual(len(responses.calls), 1)
+
+        data = json.loads(responses.calls[0].request.body)
+        self.assertIn(str(applicant), data['message'])
+        self.assertIn(str(project), data['message'])
+
+    @override_settings(
+        SLACK_DESTINATION_URL=target_url,
+        SLACK_DESTINATION_ROOM=target_room,
+    )
+    @responses.activate
+    def test_slack_staff_update_payment_request(self):
```

Comment:
Staff should be updated via slack when this happens. It was absent from the requirements added now

It is correct that applicants dont get an email when they submit though.

Reply:
Fixed and rebased.